### PR TITLE
Coverage report

### PR DESCRIPTION
**Pull request by xduh**
Add coverage report (coveralls)
Igonore getters and setters

### CODE REVIEW

**Comment by xduh** [comment]
Done

File: lib/Recipient/Recipient.php [hunk 1]
```diff
@@ -71,6 +72,7 @@ public function getId()
 
     /**
      * @return PagarMe
```
Comment (devdrops) [inline lib/Recipient/Recipient.php]: Incorrect return reference `PagarMe`.

File: lib/Recipient/Recipient.php [hunk 2]
```diff
@@ -80,6 +82,7 @@ public function getBankAccount()
     /**
      * @param PagarMe
```
Comment (devdrops) [inline lib/Recipient/Recipient.php]: Incorrect return reference.

**Comment by xduh** [comment]
Done

**Pull request merged**